services/s3/README.md

```diff
@@ -2,7 +2,7 @@
 
 # s3
 
-AWS S3 service support for [go-storage](https://github.com/beyondstorage/go-storage).
+AWS S3 service support for [go-storage](https://github.com/beyondstorage/go-storage). 
```

Comment:
There's a space after '.' .
Maybe it's useless?

Reply:
As I indicated in my commit message: `Trigger s3 test.`